connman/vpn/plugins/wireguard.c

```diff
@@ -338,6 +409,35 @@ static gboolean wg_dns_reresolve_cb(gpointer user_data)
 			info->device.name);
 
 	run_dns_reresolve(info);
+}
+
+static gboolean wg_dns_reresolve_cb(gpointer user_data)
+{
```

Comment:
NB info->reresolve_id associated with this callback is not cleared here, there and elsewhere...

Reply:
9ce8543874c54a9b936850b0bdb1d9c1bf6d6cdb I think this is now ok.